kde/kmymoney4-devel/Portfile

```diff
@@ -7,11 +7,10 @@ PortGroup           kde4    1.1
 
 fetch.type          git
 git.url             git://anongit.kde.org/kmymoney
-git.branch          485aea8e20604cf79612f62d1393fde9872c0273
+git.branch          ebeeced0715a82327ae0e2ee833cf93fc3029366
 
 name                kmymoney4-devel
-version             4.8-20161022
-revision            2
+version             4.8.1
```

Comment:
Sorry for the delay on this, I am revisiting this pull just now. Shouldn't you add a date to the version (like the previous version you are changing) because this is not a final 4.8.1 release?